### ISSUE TITLE
Shutdown/Reboot Feature Missing

### ISSUE THREAD

**Issue by NexGen-3D-Printing** [koying/SPMC#400]
Device: nVidia Shield TV 16GB

OS: Stock 3.2

Rooted: Yes (SuperSu)

SPMC Version: 16.4 and 16.4.1

I have tried the stock and developer builds of OS 3.2, rooted and non rooted, SPMC fails to enable the shutdown menu, I have disabled the lean back launcher, SPMC starts up as it should, but I have to use the "shutdown" app from the Play Store and add a favourite in order to shutdown my device, this not a good way to close Kodi as it doesn't get to shut itself down safely.

I have tried installing and wiping the system completely several times, but nothing appears to work

Does anyone have a possible fix for this? Is there a file one can edit to change this parameter? Is there a version of SPMC that is designed for Rooted devices? Have I overlooked something? Or is this a common fault?

**Comment by koying** [comment]
Where is the "su" executable on your device?

**Comment by beschoenen** [comment]
#119 

I'm also having this problem.

Shield 3.2, SPMC 16.4.1

``` bash
➜  ~ adb shell
shell@foster:/ $ which su
/su/bin/su
shell@foster:/ $ 
```

**Comment by koying** [comment]
"/su/bin/su"?
@Chainfire Do you confirm that's where supsersu put the binary, nowadays? Thanks.

**Comment by filigran** [comment]
Can confirm this. I have the same issue, and have the same su path:

```
shell@foster:/ $ which su
which su
/su/bin/su
shell@foster:/ $
```

Same setup as @beschoenen

**Comment by NexGen-3D-Printing** [comment]
Sorry for the late reply, my su is in the same location as the others:

shell@foster:/ $ which su
/su/bin/su
shell@foster:/ $ 

So far everything else appears to work quite well with SPMC + PlexBmc, just need the shutdown menu to work and the system will be perfect.

**Comment by beschoenen** [comment]
@koying I saw you made a commit for this a week ago or so, will that be in the next release?

**Comment by koying** [comment]
No, I just push random commits for fun and remove them explicitly when releasing :P

**Comment by beschoenen** [comment]
Not trying to be a dick or anything :P Just curious.
